tools/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+LEGOFY=$(pwd)/bin/legofy;
+PATH_IMAGES=$(pwd)/images/benchmark/;
+
+benchmark() {
+  local flags=$@;
+
+  # Default arguments
+  if [[  $flags = '' ]]; then
+    flags='--filter-quality=80 --pattern-quality=80 --quality=good'
```

Comment:
Ha wasn't maybe clear, but "The options taking a quality argument can be one of `fast`, `good`, `best`, and `nearest`." is applying to every `...=<quality>` option

How do you think we should reword this?